browser/css/popup.css

```diff
@@ -0,0 +1,17 @@
+body {
```

Comment:
We inherit the normal style.css but these are overrides just for the popup widget.